.organizationmap

```diff
@@ -247,6 +247,7 @@ Mirantis <contact@mirantis.com> Igor Fedotov <ifedotov@mirantis.com>
 Mirantis <contact@mirantis.com> Kiseleva Alyona <akiselyova@mirantis.com>
 Mirantis <contact@mirantis.com> Mykola Golub <mgolub@mirantis.com>
 Mirantis <contact@mirantis.com> Radoslaw Zarzynski <rzarzynski@mirantis.com>
+Mirantis <contact@mirantis.com> Victor Denisov <denisovenator@gmail.com>
```

Comment:
Minor: update your commit title to "mailmap: Victor Denisov affiliation" (seems to be the style)